app/workers/zendesk_ticket_worker.rb

```diff
@@ -6,7 +6,11 @@ def perform(ticket_options)
       GovukStatsd.increment("report_a_problem.submission_from_suspended_user")
     else
       begin
-        create_ticket(ticket_options)
+        if ticket_options["requester"]["name"] && ticket_options["requester"]["name"].length > 255
```

Comment:
The try and dig methods could be used here, but I'll leave it up to you to decide if this is a good idea or not, e.g.
```ruby
if ticket_options.dig("requester", "name").try { |name| name.length > 255 }
```